app/src/main/java/com/sunasterisk/fooddaily/ui/main/MainActivity.kt

```diff
@@ -1,24 +1,92 @@
 package com.sunasterisk.fooddaily.ui.main
 
+import android.content.Context
+import android.content.Intent
+import android.os.Parcelable
+import android.view.MenuItem
+import androidx.fragment.app.Fragment
+import com.google.android.material.bottomnavigation.BottomNavigationView
 import com.sunasterisk.fooddaily.R
+import com.sunasterisk.fooddaily.data.model.FoodDetail
 import com.sunasterisk.fooddaily.ui.base.BaseActivity
+import com.sunasterisk.fooddaily.ui.fragment.CollectionFragment
+import com.sunasterisk.fooddaily.ui.fragment.CookingFragment
 import com.sunasterisk.fooddaily.ui.fragment.HomeFragment
+import com.sunasterisk.fooddaily.utils.Constants
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.custom_action_bar.*
 
 class MainActivity : BaseActivity() {
 
     override val layoutRes: Int = R.layout.activity_main
 
+    private val bottomNavSelected = BottomNavigationView.OnNavigationItemSelectedListener { item ->
+        when (item.itemId) {
+            R.id.navigation_home -> {
+                switchScreen(item, HomeFragment())
+                true
+            }
+            R.id.navigation_collection -> {
+                switchScreen(item, CollectionFragment())
+                true
+            }
+            R.id.navigation_cooking -> {
+                switchScreen(item, CookingFragment())
+                true
+            }
+            else -> false
+        }
+    }
+
     override fun initView() {
+        createFoodsAll()
+        createFoodsDaily()
         initFragment()
+        initBottomNavigation()
+    }
+
+    private fun createFoodsAll() {
+        foodsAll = intent.getParcelableArrayListExtra(Constants.EXTRA_FOOD_LIST)
+    }
+
+    private fun createFoodsDaily() {
+        for (i in 5 until foodsAll.size) {
+            if (i % 3 == 0) {
+                foodsDaily.add(foodsAll[i])
+            }
+        }
     }
 
     override fun initPresenter() {
     }
 
+    private fun initBottomNavigation() {
+        bottomNavigation.setOnNavigationItemSelectedListener(bottomNavSelected)
+    }
+
+    private fun switchScreen(menuItem: MenuItem, fragment: Fragment) {
+        textActionBarTitle.text = menuItem.title
+        supportFragmentManager.beginTransaction()
+            .replace(R.id.frameContent, fragment)
+            .commit()
+    }
+
     private fun initFragment() {
         supportFragmentManager
             .beginTransaction()
             .add(R.id.frameContent, HomeFragment())
             .commit()
     }
+
+    companion object {
+        lateinit var foodsAll: ArrayList<FoodDetail>
+        var foodsDaily: ArrayList<FoodDetail> = ArrayList()
+
+        fun getProfileIntent(context: Context, foods: List<FoodDetail>): Intent =
```

Comment:
Tại sao lại là getProfileIntent nhỉ 
Màn main là màn profile luôn à 
```suggestion
        fun getIntent(context: Context, foods: List<FoodDetail>): Intent =
```

Reply:
chỗ này mình nhìn convention xong viết theo, không suy nghĩ gì :3